scr/nyctaxi/data_processor.py

```
@@ -0,0 +1,45 @@
+import pandas as pd
```

Ruff

`pandas` imported but unused

Remove unused import: `pandas`

(F401)